src/redux/editor/reducers.ts

```diff
@@ -48,10 +50,18 @@ export const EditorConfigReducer: Reducer<EditorConfig, EditorConfigActions> = (
       }
       saveToLocalStorage(newState)
       return newState
-    case EditorConfigActionType.SET_EDITOR_PREFERENCES:
+    case EditorConfigActionType.SET_LIGATURES:
       newState = {
         ...state,
-        preferences: (action as SetEditorPreferencesAction).preferences
+        ligatures: (action as SetEditorLigaturesAction).ligatures
+      }
+      return newState
```

Comment:
You don't save this change to local-storage.

```suggestion
      }
      saveToLocalStorage(newState)
      return newState
```

Reply:
done